src/storage/FCS.cpp

```diff
@@ -423,7 +436,7 @@ bool FB_Storage::fcs_sendRequest(FirebaseData *fbdo, struct fb_esp_fcs_req_t *re
         ret = handleResponse(fbdo);
         fbdo->closeSession();
 
-        if (Signer.getCfg()->_int.fb_file && req->requestType == fb_esp_fcs_request_type_download)
+        if (Signer.getCfg()->_int.fb_file && req->requestType == fb_esp_fcs_request_type_download && req->storageType != mem_storage_type_ota_update)

```

Comment:
It can be
if (Signer.getCfg()->_int.fb_file && req->requestType == fb_esp_fcs_request_type_download && req->req->requestType != fb_esp_fcs_request_type_download_ota)